### PR TITLE
Regenerated client for Capella V3 API

### PR DESCRIPTION
**Pull request by JamesWilkinsonCB**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -175,18 +175,10 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
 
 END OF TERMS AND CONDITIONS
 
-APPENDIX: How to apply the Apache License to your work.
+Couchbase terraform-provider-couchbasecapella
```
Comment (zain-b) [inline LICENSE:178]: How has this made it's way into here 🤔

File: README.md [hunk 2]
```diff
@@ -11,21 +13,54 @@ This API client was generated by the [OpenAPI Generator](https://openapi-generat
...
+
+Add the following in import:
+
+```golang
+import "github.com/couchbaselabs/couchbase-cloud-go-client"
+```
+
+Go Modules
+```golang
+require github.com/couchbaselabs/couchbase-cloud-go-client v1.0.0
```
Comment (zain-b) [inline README.md:30]: Let's remove the modules bit in the README template override.

When Go encounters an import of a package not provided by any module in go.mod, the go command automatically looks up the module containing that package and adds it to go.mod, using the latest version

**Review by zain-b — CHANGES_REQUESTED** [review]
A couple tiny changes

File: LICENSE (hunk 1 above)
Comment (JamesWilkinsonCB) [reply LICENSE:178]: Yep, I'll get that changed to couchbase-cloud-go-client

File: README.md (hunk 2 above)
Comment (JamesWilkinsonCB) [reply README.md:30]: Okay, I'll remove this now

**Review by zain-b — APPROVED** [review]

**Pull request merged**